### DISCUSSION TITLE
Downtime for https://searx.work

### DISCUSSION THREAD

**Discussion by ice9smart** [searxng/searx-instances#282] [Report instance downtime]
I tried to reinstall SearXNG after upgrading python, but due to issue https://github.com/searxng/searxng/issues/2111 the installation failed. I've explained the reason on https://searx.work and guided users to use other public instances.

**Comment by unixfox** [comment]
Thank you for the notification. I would highly recommend you to have some kind of backup for reverting to a working version of searxng.

Usually you can do that with docker or by creating a VM backup when you are upgrading searxng.

**Comment by unixfox** [comment]
I temporally removed your instance until you fix the issue: https://github.com/searxng/searx-instances/commit/21ea9a9a0d86ea84b7e1e455c72e6f0392c06c36

Please ask to add it back once the issue has been solved.
